scalameta/contrib/shared/src/main/scala/scala/meta/contrib/implicits/DefnExtensions.scala

```diff
@@ -0,0 +1,18 @@
+package scala.meta.contrib.instances
+
+import scala.meta.Defn
+import scala.meta._
+import scala.meta.contrib.Extract
+import scala.meta.contrib.implicits.Equality._
+
+trait DefnExtensions {
```

Comment:
The IDE autocomplete will work in the same way if moving to `ExtractExtensions`?